src/components/events/Toolbar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
this file should have been deleted. Make sure you pull from dev. There have been a bunch of changes that will cause heavy merge conflicts if you don't fix right now